packages/playwright-core/src/server/registry/index.ts

```diff
@@ -142,6 +146,8 @@ const DOWNLOAD_PATHS: Record<BrowserName | InternalTool, DownloadPaths> = {
     'mac12-arm64': 'builds/firefox/%s/firefox-mac-13-arm64.zip',
     'mac13': 'builds/firefox/%s/firefox-mac-13.zip',
     'mac13-arm64': 'builds/firefox/%s/firefox-mac-13-arm64.zip',
+    'mac14': 'builds/firefox/%s/firefox-mac-13.zip',
```

Comment:
Note: firefox-mac-13.zip gets used in all the macOS versions we support. We could rename it to firefox-mac.zip at some point - maybe now?

Reply:
Let's not :)